wp-content/themes/core/components/header/subheader/css/subheader.pcss

```diff
@@ -0,0 +1,56 @@
+/* -----------------------------------------------------------------------------
+ *
+ * Component: Subheader
+ *
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader {
+	position: relative;
+	display: flex;
+	align-items: center;
+	min-height: 260px;
+	background: var(--color-grey-light);
+	padding:  var(--spacer-20) 0;
+	color: var(--color-text);
+	margin-bottom: var(--spacer-80);
+}
+
+/* CASE: Page with featured image */
+body:not(.page-page-with-no-featured-image) .c-subheader { 
+	min-height: 440px;
+	align-items: flex-end;
+	padding:  var(--spacer-20) 0 var(--spacer-80) 0; 
+	color: var(--color-white);		
+}
+
+/* -----------------------------------------------------------------------------
+ * Subheader: Content
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader__content {
+	position: relative;
+}
+
+.c-subheader__title { 
+	margin-top: var(--spacer-20);
+}
+
+/* -----------------------------------------------------------------------------
+ * Subheader: Image component
+ * ----------------------------------------------------------------------------- */
+
+.c-subheader__media__image {
+	min-height: 440px;
+}
+
+.c-subheader__media__image:after {
+	content: '';
+	position: absolute;
+	height: 100%;
+	top: 0;
+	left: 0;
+	right: 22.5%;
+	background: linear-gradient(90deg, var(--color-neutral-90) 0%, rgba(21, 21, 21, 0.6) 63.54%, rgba(21, 21, 21, 0) 100%);
+	mix-blend-mode: normal;
+	opacity: 0.9;
+}
```

Comment:
This after selector should be a child of the image selector above. Ideally, I think this should be a separate mixin and helper class that can be applied universally (rather than specific to the subheader). @ryanurban, what are you thoughts?

Reply:
I created a mixin, 'image-overlay' . I added it to .c-image--overlay and a new class .c-image--gradient-overlay, both in the image component css.

Reply:
-> Changes about this topic need some review  @ryanurban @dpellenwood Thanks!